src/components/custom-select.tsx

```diff
@@ -9,6 +9,7 @@ interface IProps {
   items: string[];
   onSelectItem: (value: string) => void;
   HeaderIcon?: SvgIcon;
+  isHeader?: boolean;
```

Comment:
This came up in the dashboard, and I'm still not a huge fan of this pattern of passing a prop that implies that the component needs to know something about a named parent or container component within the overall layout.  I think it leads to things getting confusing and needing renaming if components are moved around or reused (like we're doing in this PR) and it makes it harder to reuse the prop without ambiguity.  So, if a button is supposed to be the color green when displayed in the header, I think it's better to pass in a `color` prop rather than an `isHeader` prop (then we can reuse the `color` prop anywhere that a color change is needed rather than having to introduce a new prop and we can remove the header component without having to rename the prop in the custom-select).  In this specific case, we only currently have ONE instance of the custom-select, and that is in the header.  I would make the default styling of the custom select match what is needed in the header so a prop isn't needed at all.  So remove this prop, remove the "topHeader" styling, and then change the default color and width of the custom-select to match what is needed in the header.  We can worry about color and width changes later with new props IF we end up needing this custom-select component again.